src/server/internalServer.h

```diff
@@ -54,7 +54,8 @@ class InternalServer {
                    bool verbose,
                    bool withTaskbar,
                    bool withLibraryButton,
-                   bool blockExternalLinks);
+                   bool blockExternalLinks,
+                   std::string indexTemplateString = "");
```

Comment:
Do we really need a default argument here ?
This is internal, so no api compatibility to keep.